src/components/Task/Drawer/Form/Form.tsx

```diff
@@ -550,6 +640,32 @@ const TaskDrawerForm = ({
                 </Button>
               </Grid>
             </Grid>
+            <Dialog
```

Comment:
I tried to mimic the current design and flow of deleting a task from mpdx.org. I think the ```Dialog``` component from MaterialUI works pretty well for this kind of confirmation popup.